OSVR-Unity/Assets/OSVRUnity/src/VRHead.cs

```diff
@@ -63,12 +62,12 @@ void Start()
             {
                 Init();
                 CatalogEyes();
-                _distortionEffect = GetComponent<OsvrDistortion>();
+/*
```

Comment:
Yes - and once the Unity plugin is ported over to use the new OSVR-Core rendering functinoality, it'll get that - this was for a Unity-specific "knob" to enable mono